frontend/src/old-pages/Layout.tsx

```diff
@@ -43,6 +114,7 @@ export default function Layout({
         content={children}
         contentType="table"
         navigation={<SideBar />}
+        breadcrumbs={<Breadcrumbs slug={pageSlug} {...slugOnClick} />}
```

Comment:
issue: I think this breaks the `onClick`, you should be able to see it for yourself by just typing the `slugOnClick` Prop differently than with `any`

In short, you are spreading a function, you should set `onClick={slugOnClick}`

Reply:
Fixed